lib/workers/repository/config-migration/branch/rebase.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  RenovateConfig,
+  defaultConfig,
+  git,
+  loadJsonFixture,
+  platform,
+} from '../../../../../test/util';
+import { GlobalConfig } from '../../../../config/global';
+import { MigratedData } from './migrated-data';
+import { rebaseMigrationBranch } from './rebase';
+
+jest.mock('../../../../util/git');
+
+describe('workers/repository/config-migration/branch/rebase', () => {
+  beforeAll(() => {
+    GlobalConfig.set({
+      localDir: '',
+    });
+  });
+
+  describe('rebaseMigrationBranch()', () => {
+    const raw = loadJsonFixture('./renovate.json');
+    const indent = '  ';
+    const renovateConfig = JSON.stringify(raw, undefined, indent) + '\n';
+    const fileName = 'renovate.json';
+
+    let config: RenovateConfig;
+    let migratedConfigData: MigratedData;
+
+    beforeEach(() => {
+      jest.resetAllMocks();
+      GlobalConfig.set({
+        dryRun: false,
```

Comment:
this overwrites all existing glbal config inclusiv `localDir`, so is `localDir` required?
If not, simply call `GlobalConfig.reset()`